tests/DCPS/Monitor/run_test.pl

```diff
@@ -80,6 +84,13 @@
 
 unlink $dcpsrepo_ior;
 
+my $mon_count = `grep -c Report mon.out`;
```

Comment:
This should be able to run on systems without 'grep'.  Equivalent functionality takes just a few lines of perl.